Document and expose a contract for using translations directly (#46)
seek-oss/vocab: README.md

extractor.addChunk(chunkName);
```

## ICU Message format

> Translation messages can sometimes contain dynamic values, such a date/time, link or username. These values can often exist somewhere in the middle of a message and change location based on translation.

Reviewer: ```suggestion
Translation messages can sometimes contain dynamic values, such as dates/times, links or usernames. These values can often exist somewhere in the middle of a message and change location based on translation.
```

To support this Vocab uses [Format.js's intl-messageformat] allowing you to use [ICU Message syntax](https://formatjs.io/docs/core-concepts/icu-syntax/) in your messages.

> In the below example we use two messages, one that passes in a single parameter and one

Reviewer: Missing the end of the sentence here.

```json
{
  "my key with param": {
    "message": "Bonjour de {name}"

>   }

Reviewer: ```suggestion
  },
```

  "my key with component": {
    "message": "Bonjour de <Link>Vocab</Link>"
  }
}
```

> Vocab will automatically parse these strings as ICU messages, identify the required parameters and ensure Typescript knows the values must be passed in.

Reviewer: ```suggestion
Vocab will automatically parse these strings as ICU messages, identify the required parameters and ensure TypeScript knows the values must be passed in.
```

Author: Thanks
